orbit/pkg/execuser/execuser_windows.go

```diff
@@ -230,3 +238,62 @@ func startProcessAsCurrentUser(appPath, cmdLine, workDir string) error {
 
 	return nil
 }
+
+// setupSyncChannel creates the synchronization channel through which child process will be
+// signalled for termination. Code is using kernel named object as the sync primitive.
+// https://learn.microsoft.com/en-us/windows/win32/sync/using-event-objects
+func setupSyncChannel(channelId string) error {
```

Comment:
So without this the child processes would end up orphans if the parent is killed?

Reply:
The purpose of setupSyncChannel() is to ensure that orbit has a way to signal the fleet-desktop app for graceful termination. We want to avoid force-killing the app as that generates the problem of duplicated icons in the traybar (fleet-desktop closing itself without unregistering the icon). 

If orbit is force killed when running on windows, two things will happen: 1) The desktop-app runner will detect that condition and will end signaling the desktop-app process for termination, and 2) the windows SCM will end up spawning a new orbit.exe process as the process is managed as a windows service, which will indirectly cause a check for orphan process during desktop-app runner initialization. See [here](https://github.com/fleetdm/fleet/blob/b3cab112725ab6789921ee679157612327a02206/orbit/cmd/orbit/orbit.go#L646)